### ISSUE TITLE
how to solve: Check failed: end < layers_.size() (-1 vs. 0)

### ISSUE THREAD

**Issue by PengfeiFang-sioc** [gnina/scripts#65]
Dear all, 
I am a beginner and trying the first time to train a model using the train.py and the default2018.model. I am receiving this error message:

I1204 12:22:43.597291  9247 solver.cpp:352] Iteration 0, Testing net (#0)
I1204 12:22:43.597334  9247 solver.cpp:352] Iteration 0, Testing net (#1)
F1204 12:22:43.597359  9247 net.cpp:523] Check failed: end < layers_.size() (-1 vs. 0) 
*** Check failure stack trace: ***

How can I solve this?
Thanks a lot!

**Comment by dkoes** [comment]
Please provide enough information to reproduce your issue (e.g. commandline and input files).

**Comment by PengfeiFang-sioc** [comment]
The commandline is:
python3 train.py -m default2018.model -p it0_tt_v1.3_0_ -d CrossDocked2020 --weights crossdock_default2018.caffemodel

I have also tried to use hires_affinity.model, the all.types on csar dataset, without the weights, and different combinations. This error will show up every time. 

This commandline will generate a solver.xxx.prototxt file, an empty traintest.xxx.prototxt file and an empty traintrain.xxx.prototxt file.

My system configuration is:
Operating system: ubuntu 20.04
Compiler: GCC 9.4.0
CUDA version (if applicable): V10.1.243
Python version: Python 3.8.10

**Comment by PengfeiFang-sioc** [comment]
I just found that the train.py I was using had problem. I downloaded the train.py again. Now that error disappeared. But I am receiving a new error.

The commandline is 

python3 train.py -m default2018.model -p it0_tt_v1.3_0_ -d CrossDocked2020 --weights crossdock_default2018.caffemodel

The error is:

  I1206 12:19:59.027737 25508 net.cpp:385] data -> rmsd_true
Traceback (most recent call last):
  File "/home/fangpengfei/gnina/scripts-train/train.py", line 934, in <module>
    results = train_and_test_model(args, train_test_files[i], outname, cont)
  File "/home/fangpengfei/gnina/scripts-train/train.py", line 441, in train_and_test_model
    solver = caffe.get_solver(solverf)
ValueError: Could not open completerec
